commcare_connect/opportunity/tables.py

```diff
@@ -63,3 +63,18 @@ class Meta:
         fields = ("user.username", "accepted")
         empty_text = "No users invited for this opportunity."
         orderable = False
+
+
+class PaymentUnitTable(tables.Table):
+    details = columns.LinkColumn(
+        "opportunity:edit_payment_unit",
+        verbose_name="",
+        text="Edit",
+        args=[utils.A("opportunity.organization.slug"), utils.A("opportunity.id"), utils.A("pk")],
+    )
+
+    class Meta:
+        model = PaymentUnit
+        fields = ("name", "amount")
```

Comment:
This should probably also show the delivery units, so the users can see what they have configured.

Reply:
Added. https://github.com/dimagi/commcare-connect/pull/165/commits/5a9b2594207aae7d452aa3c1f418cac1f5c1bad5